### ISSUE TITLE
Add PSC & Team to OCA's repo's readme.md file

### ISSUE THREAD

**Issue by ygol** [OCA/maintainer-tools#480]
It will be useful to have the PSC + team listed in the README.md file of an OCA repository.

**Comment by rousseldenis** [comment]
> It will be useful to have the PSC + team listed in the README.md file of an OCA repository.

Indeed, as github interface does not display on repository the Team involved in. IMHO it could be a link to the PSC. e.g.: https://github.com/orgs/OCA/teams/logistics-maintainers

**Comment by rousseldenis** [comment]
I can do it for the PSC I'm involved in.

**Comment by mourad-ehm** [comment]
Hi @rousseldenis, The link https://github.com/orgs/OCA/teams redirect me to https://github.com/OCA with this message "You’re not a member of any teams in this organization."
How can I find a PSC of an repository.
